order/order.go

```diff
@@ -0,0 +1,119 @@
+package order
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/cloudflare/redoctober/hipchat"
+)
+
+const (
+	NewOrder       = "%s has created an order for the label %s. requesting %s delegations for %s"
+	NewOrderLink   = "@%s - https://%s?%s"
+	OrderFulfilled = "%s has had order %s fulfilled."
+	NewDelegation  = "%s has delegated the label %s to %s (per order %s) for %s"
+)
+
+type Order struct {
+	Name string
```

Comment:
`Name` should probably be renamed to `Owner` (i.e. the user who placed the order request)
and (as @jkroll-cf suggested) a `Users  []string` field for the users that delegations are requested for (possibly empty)
`Label` should also instead be `Labels []string`